6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,55 @@
+//  Parse Arguments using process.argv
+let avg;
+function average() {
+  let sum = 0;
+  let count = 0;      
+  if(process.argv.length < 5){
```

Comment:
Good check! You could set the limit at 4, since we can also find the average between two numbers, but that's not really important 👌